.github/.syncignore

```diff
@@ -0,0 +1,4 @@
+CODEOWNERS
+workflows/create-release.yml
```

Comment:
I think we should try to enable dependabot for these three files, since they're not being synched. This will help ensure the actions all stay up-to-date.

Reply:
The plan is to add them on the github config, as these workflows are backward compatible, so they can support jammy stacks too.